src/common/redux/actions/api_key/index.ts

```diff
@@ -0,0 +1,15 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+// import * as enable from "./enable";
+import * as set from "./setKey";
+import * as removeKey from "./removeKey";
+
+export {
+    set,
```

Comment:
you need to have a stability with the action naming, note `set` and `removeKey` difference